### PR TITLE
feat(Api): Add more api endpoints to the scui api

### PR DESCRIPTION
**Pull request by BugisoftRSG**
- Send socialclub message
- Get UGC Job Data via scui api and download metadata from prod.cloud

### CODE REVIEW

**Review by Yimura — DISMISSED** [review]

File: src/views/network/view_player_database.cpp [hunk 1]
```diff
@@ -108,6 +109,21 @@ namespace big
 					session::join_by_rockstar_id(current_player.rockstar_id);
 				});
 
+				static char message[20];
```
Comment (maybegreat48) [inline src/views/network/view_player_database.cpp]: Just 20 chars?

Comment (BugisoftRSG) [reply src/views/network/view_player_database.cpp]: I could swear I looked up in gta for the max chars 255 but guess I havent replaced it thanks for pointing out

**Comment by maybegreat48** [comment]
Are you sure you've really implemented the UGC thing?

**Comment by BugisoftRSG** [comment]
> Are you sure you've really implemented the UGC thing?

Im currently on it its more a todo list. The language from the metadata file can't be queried efficently so guess I have loop over the languages if you dont know another solution. The Job Details (scui) doesnt contain it and QueryContentByContentId (QueryContent, pros.ros) neither they are also sending all languages to the server and no the response also doesnt contain it

(all tools I know about just loop over it and search for a 200 response)

**Review by Yimura — APPROVED** [review]

**Pull request merged**